charts/redpanda/values.yaml

```diff
@@ -377,7 +377,7 @@ resources:
       # If omitted, the `min` value is equal to the `max` value (requested resources defaults to limits).
       # This setting is equivalent to `resources.requests.memory`.
       # For production, use 10Gi or greater.
-      # min: 2.5Gi
+      min: 2.5Gi
```

Comment:
According to the comment on this field, it seems like it would be preferable to set the `max` and not the min. Should we do that or change the documentation?

Reply:
You are right that comment on this field is wrong and needs to be adjusted.